client/Cargo.toml

```diff
@@ -31,9 +31,9 @@ raw-window-handle = "0.5.0"
 gilrs = "0.10.1"
 
 [target.'cfg(target_os = "linux")'.dependencies]
-openxr = { version = "0.17.0", optional = true, features = ["loaded"] }
+openxr = { version = "0.17.1", optional = true, features = ["loaded"] }
```

Comment:
Random question. Should we have these locked by major.minor.sub versions? Or is major.minor version okay?